en/04_Changelogs/5.0.0.md

```diff
@@ -940,20 +940,20 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 - Removed deprecated class `SilverStripe\Dev\SapphireInfo` without equivalent functionality to replace it
 - Removed deprecated class `SilverStripe\Dev\Tasks\MigrateFileTask` without equivalent functionality to replace it
 - Removed deprecated class `SilverStripe\Dev\TestSession_STResponseWrapper` without equivalent functionality to replace it
-- Removed deprecated class `SilverStripe\Forms\GridField\GridFieldVersionedState` - use `VersionedGridFieldState` instead
+- Removed deprecated class `SilverStripe\Forms\GridField\GridFieldVersionedState` - use [`SilverStripe\Versioned\VersionedGridFieldState\VersionedGridFieldState`](api:SilverStripe\Versioned\VersionedGridFieldState\VersionedGridFieldState) instead
 - Removed deprecated class `SilverStripe\Forms\HTMLEditor\TinyMCEGZIPGenerator` without equivalent functionality to replace it
 - Removed deprecated class `SilverStripe\ORM\Connect\PDOConnector` without equivalent functionality to replace it
 - Removed deprecated class `SilverStripe\ORM\Connect\PDOQuery` without equivalent functionality to replace it
 - Removed deprecated class `SilverStripe\ORM\Connect\PDOStatementHandle` without equivalent functionality to replace it
 - Removed deprecated class `SilverStripe\ORM\Map_Iterator` without equivalent functionality to replace it
-- Removed deprecated class `SilverStripe\View\Embed\EmbedResource` - use `EmbedContainer` instead
+- Removed deprecated class `SilverStripe\View\Embed\EmbedResource` - use [`SilverStripe\View\Embed\EmbedContainer`](api:SilverStripe\View\Embed\EmbedContainer) instead
 - Removed deprecated class `SilverStripe\View\Parsers\Diff` - use [`SilverStripe\View\Parsers\HtmlDiff`](api:SilverStripe\View\Parsers\HtmlDiff) instead
 - Removed deprecated class `SilverStripe\View\Parsers\HTML4Value` - use [`SilverStripe\View\Parsers\HTMLValue`](api:SilverStripe\View\Parsers\HTMLValue) instead
 - Removed deprecated interface `SilverStripe\Control\Email\Mailer` - use `symfony/mailer` instead
-- Removed deprecated interface `SilverStripe\Control\RequestFilter` - use `HTTPMiddleware` instead
+- Removed deprecated interface `SilverStripe\Control\RequestFilter` - use [`SilverStripe\Control\Middleware\HTTPMiddleware`](api:SilverStripe\Control\Middleware\HTTPMiddleware) instead
 - Removed deprecated interface `SilverStripe\View\Requirements_Minifier` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Control\Controller::disableBasicAuth()` - add this controller's url to SilverStripe\Security\BasicAuthMiddleware.URLPatterns injected property instead
-- Removed deprecated method `SilverStripe\Control\Director::isManifestFlushed()` - use `Kernel::isFlushed` instead
+- Removed deprecated method `SilverStripe\Control\Director::isManifestFlushed()` - use [`SilverStripe\Core\Kernel::isFlushed()`](api:SilverStripe\Core\Kernel::isFlushed()) instead
```

Comment:
```suggestion
- Removed deprecated method `SilverStripe\Control\Director::isManifestFlushed()` - use [`SilverStripe\Core\CoreKernel::isFlushed()`](api:SilverStripe\Core\CoreKernel::isFlushed()) instead
```
It's not on the interface, and it can't be called from the abstract class either, so this is the correct class to reference.

Reply:
Done